src/rust/ide/view/graph-editor/src/builtin/visualization/java_script/geoMap.js

```diff
@@ -125,12 +137,19 @@ class GeoMapVisualization extends Visualization {
     initStyle() {
         let defaultMapStyle = 'mapbox://styles/mapbox/light-v9'
         let accentColor = LIGHT_ACCENT_COLOR
+        let labelBackgroundColor = LIGHT_LABEL_BACKGROUND
+        let labelColor = LIGHT_LABEL_COLOR
         if (document.getElementById('root').classList.contains('dark-theme')) {
-            defaultMapStyle = 'mapbox://styles/mapbox/dark-v9'
+            defaultMapStyle =
+                'mapbox://styles/go-find/ckis3xdqh0l8l1al2b0z41cgn' // Move to enso mapbox account, temporarily moved to mine private.
```

Comment:
Maciek, you cant keep PR "ready for review" in such a state. Please make it draft, and make it "ready for review" after this is fixed. Otherwise @farmaazon forces me tocheck it!

Reply:
now this is fixed 👍